README.md

```diff
@@ -63,6 +63,16 @@ void myBackgroundTask() {
 You will need to create a custom `MainApplication` class as follows:
 
 ```java
+//TODO: Change the package name to your app's package name
+package com.example.somepackage;
+
+import android.os.Bundle;
+import io.flutter.app.FlutterActivity;
```

Comment:
Unused import

Reply:
yeah sorry

Reply:
There is still 1 change requested after which I can merge the pull request.